garage/misc/logger.py

```diff
@@ -331,7 +338,7 @@ def stub_to_json(stub_sth):
         return {stub_to_json(k): stub_to_json(v) for k, v in stub_sth.items()}
     elif isinstance(stub_sth, (list, tuple)):
         return list(map(stub_to_json, stub_sth))
-    elif type(stub_sth) == type(lambda: None):
+    elif isinstance(stub_sth, (types.FunctionType, types.BuiltinFunctionType)):
```

Comment:
what does this do?

Reply:
This changed is required to solve [E721](https://lintlyci.github.io/Flake8Rules/rules/E721.html). However, on the left side there's an anonymous function, so we need to compare if stub_sth is an instance of function or built-in function.
I've read this does not cover all cases, and using the function callable(stub_sh) could be a better approach. What do you think Ryan?